builders/buildPost.ts

```diff
@@ -21,14 +21,16 @@ const buildPost = ($: CheerioSelector): Post => {
       .slice(0, 8)
       .join(" ") + "...";
 
-  return { title, id, timestamp, message, images };
+  console.log(link);
```

Comment:
`console.log` to remove again ;-) maybe we should add some pre-build step to check those at some point ;-)